roles/openshift_sanitize_inventory/tasks/unsupported.yml

```diff
@@ -32,3 +32,15 @@
       openshift_master_dynamic_provisioning_enabled to True and set an
       openshift_cloudprovider_kind. You can disable this check with
       'dynamic_volumes_check=False'.
+
+- name: Ensure that gluster CNS/CRS/External Gluster are configured right
```

Comment:
Rewrite this to something like: `Ensure the hosted registry's GlusterFS storage is configured correctly`